langchain4j-core/src/main/java/dev/langchain4j/agent/tool/lambda/LambdaMetaUtils.java

```diff
@@ -0,0 +1,88 @@
+package dev.langchain4j.agent.tool.lambda;
+
+import java.lang.invoke.MethodType;
+import java.lang.reflect.AccessibleObject;
+import java.lang.reflect.Method;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+
+interface LambdaMetaUtils {
+
+    static <T extends AccessibleObject> T setAccessible(T object) {
+        return AccessController.doPrivileged((PrivilegedAction<T>) () -> {
```

Comment:
You are right, I try to change JDK17, it popup the inspection.
![image](https://github.com/langchain4j/langchain4j/assets/30573693/f903f8d5-caa3-4db7-950a-0ab45602913b)

Could i mark the method with `@SuppressWarnings("removal")`, it will skip the IDEA inspection.
![image](https://github.com/langchain4j/langchain4j/assets/30573693/a744ccd5-b8a9-422c-98ee-3ab80c0534f8)

Reply:
[3c2fbc2](https://github.com/langchain4j/langchain4j/pull/136/commits/3c2fbc22cb952ebbbb3ec7f7f9fabf3ee02aa366)